test/foundry_tests/NodeELRewardVault.t.sol

```diff
@@ -1,9 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity 0.8.16;
 
-import { Test } from "forge-std/Test.sol";
-import { console2 } from "forge-std/console2.sol";
-
+import "forge-std/Test.sol";
```

Comment:
The import of the individual contracts should be preferred to importing everything as a catch all.

Reply:
Yes, I preferred individual importing of contracts personally to prevent shadow imports, in this particular case linting is failed because of `no-console`, I checked that at a lot of places we are importing everything.

Reply:
[e39a3c9](https://github.com/stader-labs/ethx/pull/235/commits/e39a3c93813985afe968cbb6329f191e9f5712d4)